HandmadeMath.h

```diff
@@ -233,7 +233,7 @@ extern "C"
 
 #define HMM_MIN(a, b) (a) > (b) ? (b) : (a)
 #define HMM_MAX(a, b) (a) < (b) ? (b) : (a)
-#define HMN_ABS(a) (a) < 0 ? -(a) : (a)
+#define HMM_ABS(a) ((a) > 0 ? (a) : -(a))
```

Comment:
The math library I was referencing with was the other way around, that's the only reason.